### PR TITLE
[Hold][WIP] Platform embedding recommender

### PR DESCRIPTION
**Pull request by Paul-Cornell**
Work in progress. UI design has changed significantly since the initial drafts.

### CODE REVIEW

File: platform/recommend.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+3. In the **Embed Recommender** pane, for **Sources**, select the source connector.
+4. Click **Start**.
+5. In the **Recommendations** pane, wait until the **Recommended Model** field displays the recommendation.
+6. To get more details, click the row in the list for your source connector.
+
+## Apply a recommendation to a custom workflow
+
+For a [Build it with me - Custom](/platform/workflows#build-it-with-me-custom) workflow:
+
+1. In the **Embed** section, for **Provider**, select the recommended embedding provider's name.
```
Comment (MKhalusova) [inline platform/recommend.mdx:29]: Can I choose models from different providers?

File: platform/recommend.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+2. If you select **OpenAI**, then for **Model**, select the recommended embedding model's name.
+3. Make a note of the number of dimensions displayed for the model that you selected. If you selected **VertexAI** instead of 
+   **OpenAI**, the number of dimensions is 768.
+4. To save these settings, continue creating or editing the workflow as usual.
+
+<Note>
```
Comment (MKhalusova) [inline platform/recommend.mdx:35]: Can I limit my choices to a dimension size? 
If I have already configured my destination, then I may already know the dimension size, and I shouldn't be choosing models with different dimensions.

File: platform/recommend.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+6. To get more details, click the row in the list for your source connector.
+
+## Apply a recommendation to a custom workflow
+
+For a [Build it with me - Custom](/platform/workflows#build-it-with-me-custom) workflow:
+
+1. In the **Embed** section, for **Provider**, select the recommended embedding provider's name.
+2. If you select **OpenAI**, then for **Model**, select the recommended embedding model's name.
+3. Make a note of the number of dimensions displayed for the model that you selected. If you selected **VertexAI** instead of 
+   **OpenAI**, the number of dimensions is 768.
```
Comment (MKhalusova) [inline platform/recommend.mdx:32]: Is the number of dimensions displayed in the UI for each model or do I have to look it up myself?

File: platform/recommend.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+4. To save these settings, continue creating or editing the workflow as usual.
```
Comment (MKhalusova) [inline platform/recommend.mdx:33]: How many models can I choose to compare? Will I be charged for this recommender selection?

**Review by MKhalusova — COMMENTED** [review]
I have several questions, though they may be partly for the docs, and partly for the functionality itself.

File: platform/recommend.mdx (hunk 1 above)
Comment (Paul-Cornell) [reply platform/recommend.mdx:29]: No. Unstructured makes its recommendation based on a limited set of pre-approved embedding models.

File: platform/recommend.mdx (hunk 3 above)
Comment (Paul-Cornell) [reply platform/recommend.mdx:32]: No. I will link to where to look it up the docs.

File: platform/recommend.mdx (hunk 4 above)
Comment (Paul-Cornell) [reply platform/recommend.mdx:33]: You cannot choose the models that Unstructured considers in making its recommendation. I will note this in the docs. 

Yes, you will be charged for this recommender selection. I will note this in the docs.

**Comment by Paul-Cornell** [comment]
Closing this PR and starting over, as the design has changed way too much since this PR was first opened.

**Pull request closed without merging**